contracts/ronin/validator/CoinbaseExecution.sol

```diff
@@ -336,39 +362,31 @@ abstract contract CoinbaseExecution is
    * Note: This method should be called once in the end of each period.
    *
    */
-  function _setNewValidatorSet(address[] memory _newValidators, uint256 _newValidatorCount, uint256 _newPeriod) private {
-    // Remove exceeding validators in the current set
-    for (uint256 _i = _newValidatorCount; _i < _validatorCount;) {
-      delete _validatorMap[_validatorIds[_i]];
-      delete _validatorIds[_i];
+  function _setNewValidatorSet(address[] memory newValidators, uint256 newPeriod, uint256 nextEpoch) private {
```

Comment:
fix comment "Note: This method should be called once in the end of each **period**."